Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CertificateItem.kt

```diff
@@ -1,5 +1,13 @@
 package de.rki.coronawarnapp.covidcertificate.person.ui.details.items
 
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
 import de.rki.coronawarnapp.util.lists.HasStableId
 
-interface CertificateItem : HasStableId
+interface CertificateItem : HasStableId {
+
+    fun CwaCovidCertificate.toStableListId(): Long = run {
+        var result = this::class.hashCode()
+        result = 31 * result + certificateId.hashCode()
+        result.toLong()
+    }
+}
```

Comment:
@d4rken Check on monday whether it would make more sense to use the `containerId` here.